blog/2022/2022-04-03-new-home-for-the-loopback-blog.md

```diff
@@ -0,0 +1,115 @@
+---
+title: New Home for the LoopBack Blog
+date: 2022-04-03
+slug: new-home-for-the-loopback-blog
+authors: achrinza
+tags: [Meta]
+---
+
+After a long hiatus, the LoopBack Blog has gotten a new home! Learn how it came
+to be and how we managed to modernise blogging stack along the way.
+
+<!--truncate-->
+
+Since the early days of LoopBack, the blog was managed by a team in StrongLoop
+(an IBM Company) under https://strongloop.com/strongblog. However, ever since
+the handover of LoopBack from IBM to the OpenJS Foundation, the state of the
+blog was in limbo; There no longer was a team to create these posts and
+although LoopBack was re-homed, StrongLoop itself was not. So the new
+maintainers were left without an existing blogging system to work with and
+focus shifted to other parts of the project that were actionable at that time.
+
+## Coming Up With a Plan
+
+Although the blog was no longer being updated, the project itself continued to
```

Comment:
Suggestion to make it a bit concise:

----
Putting the blog posts in the loopback.io repo is a natural progression, since all the documentation and text-based content will be in the same repo. The concern is it is Jekyll based. Although it worked fine….